src/puppetlabs/trapperkeeper/services_internal.clj

```diff
@@ -58,6 +58,13 @@
    :dependencies (schema/pred sequential?)
    :fns-map FnsMap})
 
+(def ProtocolandDependenciesMap
+  "Schema defining the map used to represent the protocol and dependencies
+  of a service."
+  {:fns [clojure.lang.PersistentList]
```

Comment:
ok, but is there something about using the Java class here that is compelling to you?  Like... does it catch/enforce something that we'd miss if we did something more clojure-y?  I just want to make sure I'm not overlooking something that you thought of.